UPGRADE-2.0.md

```diff
@@ -121,6 +121,9 @@
 | `sylius.active_promotions_provider`                                                                        | `sylius.provider.active_promotions`                                                  |
 | `sylius.promotion_coupon_generator`                                                                        | `sylius.generator.promotion_coupon`                                                  |
 | `sylius.promotion_coupon_generator.percentage_policy`                                                      | `sylius.generator.percentage_generation_policy`                                      |
+| **TaxationBundle**                                                                                         |                                                                                      |
+| `sylius.tax_rate_resolver`                                                                                 | `sylius.resolver.tax_rate`                                                           |
+| `sylius.tax_rate_date_eligibility_checker`                                                                 | `sylius.eligibility_checker.tax_rate_date`                                           |
```

Comment:
```suggestion
| `sylius.tax_rate_date_eligibility_checker`                                                                 | `sylius.checker.tax_rate_date_eligibility`                                           |
```
or even
```suggestion
| `sylius.tax_rate_date_eligibility_checker`                                                                 | `sylius.checker.tax_rate.date_eligibility`                                           |
```
?

Reply:
We should discuss that and possibly change that in a separate PR for all eligibility checkers 🖖🏻